tau2_integration/saga_workflow_builder.py

```
@@ -0,0 +1,410 @@
+                            'tool': tool_name,
+                            'args': tool_args,
+                            'result': result,
+                            'timestamp': datetime.now().isoformat()
+                        })
+                        
+                        tool_messages.append(ToolMessage(
+                            content=str(result),
+                            tool_call_id=tool_call['id']
+                        ))
+                        
+                    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)